third_party/sl4a/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # This version code implements the versioning recommendations in:
 # https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
-VERSION_CODE=1000100
-VERSION_NAME=1.0.1-SNAPSHOT
+VERSION_CODE=1000199
```

Comment:
99 is the RC name of the final release for a particular version code. You should read the blog link mentioned in the comment right above this line :)